CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
```

Comment:
```suggestion

:tada::raised_hands: First, thanks for taking the time to read this and consider to contribute! :raised_hands::tada:

Contributions are welcome, and they are greatly appreciated.

```

The first sentence is part of the atom https://github.com/atom/atom/blob/master/CONTRIBUTING.md file.
I use the suggested 2 lines in https://github.com/lotharschulz/ktorjib/blob/main/CONTRIBUTING.md to welcome contributions.